.github/workflows/tests.yml

```diff
@@ -60,11 +60,23 @@ jobs:
       - name: Install Yarn
         run: npm install -g yarn
       # Build and install the package if it's Windows, to service shell-based tests
-      - if: ${{ matrix.os }} == "windows-latest"
+      - name: Install binary on Windows
+        if: ${{ matrix.os == 'windows-latest' }}
+        env:
+          SKIP_PRE_BUILD: true
+        # Make `locust` available on the Windows shell by installing it with `pip`
+        # before running tests which invoke it in the `cmd` prompt
         run: |
           python -m poetry build
-          pip install --find-links=dist locust
-      - run: python -m poetry run tox -e ${{ matrix.tox }}
+          python -m poetry self add "poetry-plugin-export"
+          python -m poetry export --without-hashes --format=requirements.txt > requirements.txt
+          pip install -r requirements.txt
+          pip install poetry-core>=1.0.0
+          pip install locust --find-links=dist
+      - name: Run tox tests
+        env:
+          SKIP_PRE_BUILD: ${{ matrix.skip_pre_build }}
```

Comment:
Will the string "false" really be evaluated as falsey? I would think you would have to do something such as `${{ matrix.skip_pre_build }} == "true"` in order for the statement to be truthy or falsey

Reply:
In this case, the thing that checks the string is in `pre_build.py`:

```
os.environ.get("SKIP_PRE_BUILD", "") == "true"
```

Passing a boolean would be preferably, but you'd need to play cat and mouse between OS versions and how ENV vars are handled with respect to `"true" | True | true` etc. which is normally fairly painful

I could try and get this working without using the string if you want 🤔

Reply:
No I don't have a problem with the strings I was just checking that it wasn't something that was missed since I didn't see any `== "true"` in the diff